src/libstd/alloc.rs

```diff
@@ -133,61 +134,101 @@ pub use alloc_crate::alloc::*;
 #[derive(Debug, Default, Copy, Clone)]
 pub struct System;
 
-// The AllocRef impl checks the layout size to be non-zero and forwards to the GlobalAlloc impl,
-// which is in `std::sys::*::alloc`.
```

Comment:
Why remove this comment? It's still forwarding to the `GlobalAlloc` impl.

Reply:
That was by accident, thanks for catching this!